dev/driver/run_JWAFS_GCIP.wcoss2

```diff
@@ -1,10 +1,10 @@
-#!/bin/sh
+#!/bin/bash
 
 #PBS -j oe
 #PBS -o /lfs/h2/emc/ptmp/yali.mao/working_wafs/log.wafs_gcip
 #PBS -N wafs_gcip
 #PBS -l walltime=00:30:00
-#PBS -l select=1:ncpus=2:mem=50GB
+#PBS -l select=1:ncpus=1
```

Comment:
It seems you will need 25GB for this job, based on the 50GB that was used before.